submissions/xT0PERx/Memory pair game/transitions.js

```diff
@@ -0,0 +1,116 @@
+const imageses = [
+  'images/Alfa-romeo.jpeg',
+  'images/Ford.jpeg',
+  'images/Mazda.jpeg',
+  'images/Nissan.jpeg',
+  'images/Peugeot.jpeg',
+];
+//my random array with my img
+const cardDeck = imageses.concat(imageses);
```

Comment:
Also, you can to use [spread](http://jsraccoon.ru/es6-spread-rest) 
` const cardDeck = [...imageses, ... imageses];`